dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/ZenodoHelper.java

```diff
@@ -101,6 +103,13 @@ public final class ZenodoHelper {
     private static String zenodoClientID;
     private static String zenodoClientSecret;
 
+    /**
+     * A rate limit helper that waits for up to 61 seconds for the limit to be reset. Some reset should happen every 60 seconds, add
+     * a 1 second padding.
+     */
+    private static final ClientRateLimitHelper RATE_LIMIT_HELPER = new ClientRateLimitHelper(Duration.ofSeconds(61));
```

Comment:
I wonder if we should instantiate a `ClientRateLimitHelper` at the beginning of each call to `findDoisForGitHubRepo`, rather than once and shared.  Right now, this code works because there's no state in `ClientRateLimitHelper` for multiple threads to mangle, but I can imagine us adding state to `ClientRateLimitHelper` someday...

Reply:
Changed it. I wonder if it should just be static method instead, but this can let it grow into something with state if needed.